src/core/public/application/utils/parse_app_url.test.ts

```diff
@@ -6,8 +6,8 @@
  * Side Public License, v 1.
  */
 
-import { App } from '../types';
-import { BasePath } from '../../http/base_path';
+import type { App } from '../types';
+import { BasePath } from '@kbn/core-http-browser-internal';
```

Comment:
nit: absolute exports should be on top (for some reason, ESLint does not apply the fix in test files)